### PR TITLE
feat: redesign header&bar

### PR DESCRIPTION
**Pull request by Senna46**
It was divided into three paths, each redesigned.

1. utilities
2. nft-backed-loan
3. derivatives

Test 

- [ ] Routing
- [ ] Tx Hash window

スクリーンショット 2023-03-28 195730
スクリーンショット 2023-03-28 204157

### CODE REVIEW

**Comment by Senna46** [comment]
close #415
Sidebar always open on laptops and above

**Comment by mkXultra** [comment]
https://github.com/UnUniFi/lab-test/issues/86

File: projects/portal/src/app/views/tools/app-tool/app-tool.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  selector: 'view-app-tool',
+  templateUrl: './app-tool.component.html',
+  styleUrls: ['./app-tool.component.css'],
+})
+export class AppToolComponent implements OnInit {
+  apps: { name: string; link: string; icon: string }[];
+
+  constructor() {
+    this.apps = [
+      { name: 'Utilities', link: '/', icon: 'account_balance_wallet' },
```
Comment (KimuraYu45z) [inline projects/portal/src/app/views/tools/app-tool/app-tool.component.ts:13]: `assistant` is better for icon

https://fonts.google.com/icons?selected=Material+Symbols+Outlined:assistant:FILL@0;wght@400;GRAD@0;opsz@48

File: projects/portal/src/app/views/tools/app-tool/app-tool.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      { name: 'NFT Backed Loan', link: '/nft-backed-loan', icon: 'photo_library' },
```
Comment (KimuraYu45z) [inline projects/portal/src/app/views/tools/app-tool/app-tool.component.ts:14]: `loyalty` is better for icon I think
https://fonts.google.com/icons?selected=Material+Symbols+Outlined:loyalty:FILL@0;wght@400;GRAD@0;opsz@48

NFT is not only for JPEG

**Review by KimuraYu45z — CHANGES_REQUESTED** [review]

**Review by KimuraYu45z — APPROVED** [review]
LGTM

**Pull request merged**